packages/azure-themes/jest.config.js

```diff
@@ -0,0 +1,14 @@
+let { createConfig } = require('../../scripts/tasks/jest-resources');
+let path = require('path');
+
+const config = createConfig({
+  setupFiles: [path.resolve(path.join(__dirname, 'config', 'tests.js'))],
+
+  moduleNameMapper: {
+    'office-ui-fabric-react/lib/(.*)$': 'office-ui-fabric-react/lib-commonjs/$1'
+  },
+
+  snapshotSerializers: [path.resolve(__dirname, './node_modules/@uifabric/jest-serializer-merge-styles')]
+});
+
+module.exports = config;
```

Comment:
I don't think you need this unless you're planning on adding executable code that you'll want to have unit tests run against. See fluent-theme's [`jest.config.js`](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/fluent-theme/jest.config.js). Also, `fluent-theme` does not have a `config/tests.js` file.